app/views/customers/index.html.erb

```diff
@@ -2,9 +2,21 @@
 
 <h1>Customers</h1>
 <div class="search-menu">
-  <%= form_tag customers_path, method: :get do %>
+  <%= form_tag customers_path, method: :get, remote: true do %>
     <div>
-      <%= select_tag :game, options_from_collection_for_select(Customer::game_names, :first, :last), 
+      <script>
+        $( function() {
+          $( "#start_date" ).datepicker();
```

Comment:
Also, inside .datepicker([PUT_DEFAULT_STUFF]); ... check documentation on what you can put in there though not 100% sure names and such